lib/jetstream/api/kv.ex

```diff
@@ -151,6 +152,56 @@ defmodule Jetstream.API.KV do
     end
   end
 
+  @doc """
+  Get all the non-deleted key-value pairs for a Bucket
+
+  ## Examples
+
+      iex>%{"key1" => "value1} = Jetstream.API.KV.contents(:gnat, "my_bucket")
+  """
+  @spec contents(conn :: Gnat.t(), bucket_name :: binary()) :: %{}
+  def contents(conn, bucket_name) do
```

Comment:
This raises when Gnat.sub or Consumer.create fails. Should this be a bang function?

```suggestion
  @spec contents!(conn :: Gnat.t(), bucket_name :: binary()) :: %{}
  def contents!(conn, bucket_name) do
```

Reply:
I think this does point out that the `contents` function should return an error tuple as well if either of the underlying functions fail. I'll refactor to include that

Reply:
https://github.com/mmmries/jetstream/pull/64/commits/0af6c3ce3ac3bbef8a9019b23061ea47d0f0cdc1